vnext/Microsoft.ReactNative/Modules/NetworkingModule.cpp

```diff
@@ -229,11 +229,16 @@ void NetworkingModule::NetworkingHelper::OnRequestError(int64_t requestId, std::
 void AttachContentHeaders(
     winrt::Windows::Web::Http::IHttpContent content,
     winrt::Windows::Web::Http::Headers::HttpMediaTypeHeaderValue contentType,
-    std::string contentEncoding) {
+    std::string contentEncoding,
+    std::string contentLength) {
   if (contentType != nullptr)
     content.Headers().ContentType(contentType);
   if (!contentEncoding.empty())
     content.Headers().ContentEncoding().ParseAdd(Microsoft::Common::Unicode::Utf8ToUtf16(contentEncoding));
+  if (!contentLength.empty()) {
+    auto contentLengthHeader = atoi(contentLength.c_str());
+    content.Headers().ContentLength() = contentLengthHeader;
```

Comment:
it's interesting that this works (because ContentLength returns an IReference), but consider writing this as all the other idiomatic cppwinrt in this file:

```suggestion
    content.Headers().ContentLength(contentLengthHeader);
```